testing_and_setup/compass/ocean/scripts/plots/plot_initial_state.py

```diff
@@ -29,10 +29,11 @@ def main():
     args = parser.parse_args()
 
     # load mesh variables
-    ncfile = Dataset(args.input_file_name, 'r')
-    nCells = ncfile.dimensions['nCells'].size
-    nEdges = ncfile.dimensions['nEdges'].size
-    nVertLevels = ncfile.dimensions['nVertLevels'].size
+    chunks = {'nCells': 32768, 'nEdges': 32768}
+    ds = xarray.open_dataset(args.input_file_name, chunks=chunks)
```

Comment:
This breaks the data set into "chunks" of 32k cells or edges per "chunk".  Chunks are distributed across multiple threads (36 on a Grizzly compute node) whenever a computation like `min()`, `max()`, `xarray.plot.hist()`, etc. is performed.